amplify/backend/function/fetchMotionState/src/utils.js

```diff
@@ -33,9 +34,22 @@ const getVotingClient = async (colonyAddress) => {
   return colonyClient.getExtensionClient(Extension.VotingReputation);
 };
 
-const getLatestMotionState = async (colonyAddress, motionId) => {
-  const votingReputationClient = await getVotingClient(colonyAddress);
-  return votingReputationClient.getMotionState(motionId);
+const getLatestMotionState = async (
+  colonyAddress,
+  { nativeMotionId, createdBy },
+) => {
+  let motionState = MotionState.Null;
+  try {
+    const votingReputationClient = await getVotingClient(colonyAddress);
+    /* Only fetch state for a motion that was created by the current installation of the voting rep extn */
```

Comment:
This ensures we don't get clashes whereby two motions of native Id 1 from different installations have the same status. 

One should be invalid (the uninstalled one), and the other whatever is the current state of the motion.